src/App.js

```diff
@@ -1,7 +1,33 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import fetchCapitalCities from "./context/fetchCapitalCities";
 
-function App() {
-  return <div>App</div>;
+function App(response) {
+  const [capitals, setCapitals] = useState();
+
+  useEffect(() => {
+    fetchCapitalCities().then((res) => {
+      setCapitals(res);
+    });
+  }, []);
+
+  let capital = "";
+  if (!capitals) capital = "London";
+
+  const i = Math.trunc(Math.random() * capitals?.data.length);
+  capital = capitals?.data[i].capital;
+
+  const handleClick = () => {
+    fetchCapitalCities().then((res) => {
+      setCapitals(res);
+    });
+  };
```

Comment:
Okay, this is a bit confusing. Why are we fetching the same capital cities on every button click? What happens now is:

- You fetch cities on the first `useEffect` initially. This gets 250 capital cities.
- Then you choose a random one, display it
- When button is clicked, you fetch the exact same 250 capital cities again with the same API request
- Then you choose a random one, and display it.

Instead of making duplicate requests on every click, can we just pick a different capital on every click (using the initial capital list)? E.g. change `i` on every click rather than changing `capitals`

Reply:
Yes, you are right. Will make these changes now.

Reply:
@ozcanonur 
Done. Can you check again? 

Also, I added the country name onto the screen.